src/Compilers/CSharp/Portable/Symbols/ObsoleteAttributeHelpers.cs

```diff
@@ -190,7 +190,7 @@ static DiagnosticInfo createObsoleteDiagnostic(Symbol symbol, BinderFlags locati
                     Debug.Assert(!data.IsError);
 
                     // Provide an explicit format for fully-qualified type names.
-                    return new CustomObsoleteDiagnosticInfo(MessageProvider.Instance, (int)ErrorCode.WRN_Experimental, data,
+                    return new CustomObsoleteDiagnosticInfo(MessageProvider.Instance, (int)ErrorCode.WRN_Experimental, effectiveSeverity: DiagnosticSeverity.Error, data,
```

Comment:
@jaredpar Here's the core of the change for this design (being discussed in email). When producing WRN_Experimental, we're elevating the effective severity (before applying any of the command-line or editor config rules).